src/functions/list.js

```diff
@@ -12,18 +12,24 @@ const penaltyDocuments = new PenaltyDocument(
 );
 
 export default (event, context, callback) => {
-	let offset = 'undefined';
+	let offset = 0;
 	let exclusiveStartKey = 'undefined';
 
 	if (event.queryStringParameters != null && event.queryStringParameters.Offset !== undefined) {
 		offset = event.queryStringParameters.Offset;
 	}
 
 	if (event.queryStringParameters != null &&
-		event.queryStringParameters.ExclusiveStartKey !== undefined) {
-		exclusiveStartKey = event.queryStringParameters.ExclusiveStartKey;
+		typeof event.queryStringParameters.NextID !== 'undefined') {
```

Comment:
Double negation would make this type of statement look a bit neater:
```!!event.queryStringParameters.NextID``` This casts the expression to a Boolean value that's dependent on the truthiness of the the NextID property.